src/components/editable-input/editable-input.jsx

```diff
@@ -7,14 +7,15 @@ import styles from './editable-input-styles';
 import { Button, Icon, Input, Spinner } from '../..';
 import HelpText from '../input/help-text';
 
+const SENSITIVE_PLACEHOLDER_VALUE = '****';
```

Comment:
Just an idea that struck me: Maybe we should allow the user to pass in a value to have as the placeholder, but then just default back to this one. What do you think? :)

Reply:
I added it